src/components/Amount.tsx

```diff
@@ -11,47 +11,23 @@ function prettyPrintAmount(n?: number | bigint): string {
     return n.toLocaleString();
 }
 
-export function Amount(props: {
+export function AmountSats(props: {
     amountSats: bigint | number | undefined;
-    showFiat?: boolean;
     loading?: boolean;
-    whiteBg?: boolean;
-    align?: "left" | "center" | "right";
+    classes?: string;
     icon?: "lightning" | "chain" | "plus" | "minus";
-    size?: "small" | "large" | "xl";
-    green?: boolean;
+    denominationSize?: "sm" | "lg" | "xl";
 }) {
-    const [state, _] = useMegaStore();
-
-    const amountInUsd = () =>
-        satsToUsd(state.price, Number(props.amountSats) || 0, true);
-
     return (
-        <div
-            class="flex flex-col gap-1"
-            classList={{
-                "items-start": props.align === "left",
-                "items-center": props.align === "center",
-                "items-end": props.align === "right"
-            }}
-        >
+        <div class={`flex flex-col gap-1 ${props.classes}`}>
```

Comment:
this string interpolation sadly doesn't work. it works _sometimes_ with tailwind, but not often enough to be reliable (due to the compile-time step of tailwind, basically it could fail to compile a variant and so at runtime you don't actually have all options available)

that's why I end up using `classList` a lot: https://www.solidjs.com/docs/latest/api#classlist

Reply:
bummer ok, i'll make this change

Reply:
I went ahead and moved all classes up one level so they are not passing the classes as props they just exist as the parent elements of amountsats and amountfiat now.